### ISSUE TITLE
Framegrabber not starting with udp stream from ffmpeg cmd line

### ISSUE THREAD

**Issue by cagneymoreau** [bytedeco/javacv#1340]
Really stuck on why the following code isnt working. I recently loaded this year old project from an old hard drive into intellij on windows. downloaded the javacv-platform 1.5.2 and it compiles and runs. However grabber.start() doesn't return when I send a udp stream even through my socket and buffer show data being sent to the input stream

this is how im sending the video
ffmpeg -re -i 11.mp4 -vcodec libx264 -acodec copy -f h264 "udp://192.168.1.5:8550"

receiving code

`public class ImageDecoder {

    private final static String TAG = "ImageDecoder ";

    private ImageDecoder() {

    }

    public static void streamImageToImageView(
            final ImageView view,
            final InputStream inputStream,
            //final File file,
            final String format,
            final int frameRate,
            final int bitrate,
            final String preset,
            final int numBuffers
    ) {
        System.out.println("Image Decoder Starting...");

        try {

            final FrameGrabber grabber = new FFmpegFrameGrabber(inputStream);
            final Java2DFrameConverter converter = new Java2DFrameConverter();

            //grabber.setFrameNumber(frameRate);
            grabber.setFormat(format);
            //grabber.setVideoBitrate(bitrate);
            //grabber.setVideoOption("preset", preset);
            //grabber.setNumBuffers(numBuffers);

            System.out.println("Image Decoder waiting on grabber.start...");
            grabber.start();

            System.out.println("Image Decoder Looping----------------------------------- hit stop");
            while (!Thread.interrupted()) {
                //System.out.println("Image Decoder Looping");
                final Frame frame = grabber.grab();
                if (frame != null) {
                    //System.out.println("success frame");
                    final BufferedImage bufferedImage = converter.convert(frame);
                    if (bufferedImage != null) {
                        //System.out.println("success buff image");
                        Platform.runLater(() -> view.setImage(SwingFXUtils.toFXImage(bufferedImage, null)));
                    } else {
                        System.out.println("no buf im");
                    }
                } else {
                    System.out.println("no fr");
                    Thread.currentThread().interrupt();
                }

            }

        } catch (Exception e) {
            System.out.print(TAG + e);
        }

    }

}`

I have another class that writes the ffmpeg stream sent to a text file and it looks in order so Im really thinking its the receiving side

**Comment by saudet** [comment]
Make sure to set maximumSize to 0 to get the previous default behavior.

**Comment by saudet** [comment]
Duplicate of #1304
